input/groups/weapon.json

```diff
@@ -23,7 +23,7 @@
         3869,
         3871
       ],
-      "Ascalon Region: Steam": [
+      "Region: Ascalon / Cultural: Charr Tier 1 - Steam": [
```

Comment:
Happy with the inversion of Region. Only being in one category is a limitation I want to stick with, and I feel the region is the category to use rather than the cultural. That is, a player is more likely to acquire the skin via random drops in the region than explicitly seek it out from a cultural vendor - this is also true for some of the skins that are both regional and in dungeon vendors.